docs/source/dev_workflow.rst

```diff
@@ -5,7 +5,7 @@ Developer Workflow
 This page describes how to install and improve the development version of Blaze.
 
 If this documentation isn't sufficiently clear or if you have other questions
-then please email blaze-dev@continuum.io.
+then please email blaze-dev@anaconda.com.
```

Comment:
does this address actually reach any developers?

Reply:
I think this is the list: https://groups.google.com/a/anaconda.com/forum/?hl=en#!members/blaze-dev

Reply:
Looks like 5 threads this year, most recently in July. https://groups.google.com/a/anaconda.com/forum/#!forum/blaze-dev